### PR TITLE
Daniel d/tsk 3879 teamcity integration

### PR DESCRIPTION
**Pull request by DanielDinsh**

### CODE REVIEW

File: plugins/teamcity/mask.yaml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  name: teamcity
+  icon_uri: https://app.dev.blinkops.com/assets/images/Icons/Companies/Company/XL/TeamCity.svg
+  active: true
+is_connection_optional: false
+request_url_conn_type: teamcity
+actions:
+  getAllUsers:
+    parameters:
+      fields:
+        alias: Fields
```
Comment (ozhey) [inline plugins/teamcity/mask.yaml:30]: The `getDisplayName` function runs for all parameters automatically, so it's redundant to explicitly add an alias if it's the same (this is true for all params here)
But it's not harmful so it's fine

File: plugins/teamcity/teamcity.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	blink_conn "github.com/blinkops/blink-sdk/plugin/connections"
+	"net/http"
+)
+
+type Plugin struct{}
+
+func (p Plugin) HandleAuth(req *http.Request, conn map[string]string) error {
+	prefixes := connections.HeaderValuePrefixes{"AUTHORIZATION": consts.BearerAuthPrefix}
+	aliases := connections.HeaderAlias{"TOKEN": "AUTHORIZATION"}
+	req.Header.Set(consts.AcceptTypeHeader, consts.JsonContentType)
```
Comment (ozhey) [inline plugins/teamcity/teamcity.go:16]: can you comment why it's required? zoom requires it for get requests too?

**Review by ozhey — APPROVED** [review]

File: plugins/teamcity/teamcity.go (hunk 2 above)
Comment (DanielDinsh) [reply plugins/teamcity/teamcity.go:16]: It's TeamCity plugin, There default is XML content as a response.

File: plugins/teamcity/mask.yaml (hunk 1 above)
Comment (DanielDinsh) [reply plugins/teamcity/mask.yaml:30]: If I remember right, when we don't add the alias property it's assigned an empty string - "".

See the condition I attached as an image, and I asked someone and they told me this is in known and you know why the alias property is not in if-else condition like all other properties.

Line 326:
Screen Shot 2022-05-30 at 10 51 31

**Pull request merged**